src/db/model/stardust.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_message_stardust as stardust;
+use serde::{Deserialize, Serialize};
+
+use super::Model;
+
+/// Model for the [`Message`].
+#[derive(Serialize, Deserialize)]
+pub struct Message {
```

Comment:
We should probably go ahead and include optional metadata.

Reply:
All of these: https://github.com/iotaledger/inx/blob/develop/proto/inx.proto#L128-L153?

Reply:
Yeah, except maybe parents because that's on the message too.